ToolBox/TradingEconomicsDataDownloader/TradingEconomicsCalendarDownloader.cs

```diff
@@ -136,8 +135,8 @@ public override bool Run()
 
                     foreach (var calendarDataByDate in kvp.GroupBy(x => x.LastUpdate.Date))
                     {
-                        var date = calendarDataByDate.Key.ToString("yyyyMMdd");
-                        var tempPath = Path.Combine(Path.GetTempPath(), $"{Guid.NewGuid()}.json");
+                        var date = calendarDataByDate.Key.ToStringInvariant("yyyyMMdd");
+                        var tempPath = Path.Combine(Path.GetTempPath(), $"{Guid.NewGuid().ToStringInvariant(null)}.json");
```

Comment:
Any thoughts about making the `format` argument of `ToStringInvariant()` optional, defaulting to `null`?